app/components/newsletter/newsletter-header-overview.js

```diff
@@ -1,154 +1,180 @@
-// TODO: octane-refactor
-/* eslint-disable ember/no-get */
-// eslint-disable-next-line ember/no-classic-components
-import Component from '@ember/component';
-import { inject as service } from '@ember/service';
-import { computed } from '@ember/object';
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
 import moment from 'moment';
 import SendingOldCampaignError from 'frontend-kaleidos/errors/sending-old-campaign-error';
-
-// TODO: octane-refactor
-// eslint-disable-next-line ember/no-classic-classes, ember/require-tagless-components
-export default Component.extend({
-  intl: service(),
-  store: service(),
-  session: service(),
-  router: service(),
-  toaster: service(),
-  newsletterService: service(),
-  currentSession: service(),
-
-  agenda: null,
-  meeting: null,
-  isVerifying: null,
-
-  shouldShowPrintButton: computed('router.currentRouteName', function() {
-    return this.router.currentRouteName.includes('newsletter.print');
-  }),
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+/**
+ * @argument {Meeting} meeting
+ * @argument {Agenda} agenda
+ */
+export default class NewsletterHeaderOverviewComponent extends Component {
+  @service intl;
+  @service store;
+  @service router;
+  @service toaster;
+  @service newsletterService;
+  @service currentSession;
+
+  @tracked isVerifying = false;
+  @tracked isLoading = false;
+
+  @tracked mailCampaign;
+  @tracked newsletterHtml = null;
```

Comment:
Deze lijn klopte niet, dit is de enige ref dat zo noemt, de andere refs hier en in template zijn `newsletterHTML`